play.pokemonshowdown.com/src/battle-team-editor.tsx

```diff
@@ -854,6 +855,17 @@ class TeamTextbox extends preact.Component<{ editor: TeamEditorState, onChange?:
 				// make sure it's still there:
 				const valueIndex = this.textbox.value.indexOf(value);
 				this.replace(paste.paste.replace(/\r\n/g, '\n'), valueIndex, valueIndex + value.length);
+        const notes = paste["notes"] as string;
+				if (notes.startsWith("Format: ")) {
+					const formatid = toID(notes.slice(8));
+					if (BattleFormats[formatid]) {
+						this.editor.setFormat(formatid);
+					}
+				}
+				const title = paste["title"] as string;
+				if (title && !title.startsWith('Untitled')) {
+					this.editor.team.name = title.replace(/[\|\\\/]/g, '');
```

Comment:
tbh there's an interesting question of, what if this team already has a name, which should win? Let's poll it.